hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AInstrumentation.java

```diff
@@ -634,46 +655,56 @@ public void close() {
 
   /**
    * Statistics updated by an input stream during its actual operation.
-   * These counters not thread-safe and are for use in a single instance
-   * of a stream.
+   * These counters are marked as volatile so that IOStatistics on the stream
```

Comment:
I think this Javadoc isn't updated to the new change.

Reply:
fixed